Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/SubmissionsController.cs

```diff
@@ -330,22 +334,51 @@ public JsonResult GetSubmissionTypes(int problemId, bool? allowBinaryFilesUpload
             return this.Json(submissionTypesSelectListItems, JsonRequestBehavior.AllowGet);
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [OverrideAuthorization]
```

Comment:
What's the point of having both Authorize and OverrideAuthorization attributes on a single action?